app/javascript/listings/listings.jsx

```diff
@@ -83,10 +77,18 @@ export class Listings extends Component {
       ({ slug } = openedListing);
       document.body.classList.add('modal-open');
     }
+
+    t.debouncedClassifiedListingSearch = debounce(
+      this.handleQuery.bind(this),
+      150,
+      {
+        leading: true,
+      },
+    );
```

Comment:
Debouncing, in general, will eventually be refactored out into a more reusable method https://github.com/thepracticaldev/dev.to/pull/6358#discussion_r385807411

Reply:
IT'S HAPPENING!

Reply:
Although this seems formatted in a really weird way to me? 🤔 did your editor do this?

Reply:
I did it 🙆‍♂️. I like the it reads and it makes future code diffs very easy if we add more options to debounce (unlikely).